### PR TITLE
[ MODCAT-16 ] Authority source management.

### PR DESCRIPTION
**Pull request by natasciab**

### CODE REVIEW

File: src/main/java/org/folio/rest/impl/AuthoritySourceAPI.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ *
+ * @author natasciab
+ * @since 1.0
+ */
+
+public class AuthoritySourceAPI implements AuthoritySourcesResource {
+
+    protected final Log logger = new Log(AuthoritySourceAPI.class);
+
+    // This is the adapter that converts existing value objects (authority source in this case)
```
Comment (agazzarini) [inline src/main/java/org/folio/rest/impl/AuthoritySourceAPI.java]: @natasciab please remove these inline comments

File: src/main/java/org/folio/rest/impl/AuthoritySourceAPI.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    // in OKAPI resources.
+    private Function<ValueLabelElement, AuthoritySource> toAuthoritySource = source -> {
+        final AuthoritySource authoritySource = new AuthoritySource();
+        //TODO: handle type Integer for value element or null value in Integer.parseInt
```
Comment (agazzarini) [inline src/main/java/org/folio/rest/impl/AuthoritySourceAPI.java]: @natasciab please remove these inline comments

**Comment by carmentrazza** [comment]
I'm not enabled for the Merge Pull Request

**Comment by agazzarini** [comment]
The pullrequest contains some change request. Once everything will be 
fixed I'll merge that

**Pull request merged**